doc/en/user/source/styling/sld/working.rst

```diff
@@ -20,12 +20,17 @@ Catalog Styles
 Styles in the catalog can be viewed, edited and validated via the :ref:`styling_webadmin_styles` menu of the :ref:`web_admin`. 
 They may also be created and accessed via the REST :ref:`rest_api_styles` API.
 
-Catalog styles consist of a :ref:`sld_reference_sld` document 
+There are two types of Catalog Styles: Layer Styles (the default) and Style Groups.
+
+Layer Styles
+~~~~~~~~~~~~
```

Comment:
Consider rewriting to focus on symbology encoding document, or how an SLD can be used in this role (short listing only the first named layer lement).

> Symbology Encoding Style
> 
> Symbology Encoding style consists of an :file:`Symbology Encoding` document used to specify the styling of a single layer.
> 
> GeoServer supports the use of a :ref:`sld_reference_sld` document containing a single ``<NamedLayer>`` element, which contains a single ``<UserStyle>`` element to specify the styling. 
> 
> * When used in this fashion the layer name is ignored, since the style may be applied to many different layers.
> * When using an an :ref:`sld_reference_sld` generated by another application keep in mind only the first ``<NamedLayer>`` is used, any subsequent content is ignored.